NorthstarLauncher/main.cpp

```diff
@@ -419,19 +419,14 @@ int main(int argc, char* argv[])
 	{
 		PrependPath();
 
-		if (!fs::exists("ns_startup_args.txt"))
+		// NOTE [Fifty]: Functionality removed in https://github.com/R2Northstar/NorthstarLauncher/pull/517
+		if (fs::exists("ns_startup_args.txt"))
 		{
-			std::ofstream file("ns_startup_args.txt");
-			std::string defaultArgs = "-multiple";
-			file.write(defaultArgs.c_str(), defaultArgs.length());
-			file.close();
+			std::cout << "[*] WARNING: 'ns_startup_args.txt' is no longer supported!" << std::endl;
 		}
-		if (!fs::exists("ns_startup_args_dedi.txt"))
+		if (fs::exists("ns_startup_args_dedi.txt"))
 		{
-			std::ofstream file("ns_startup_args_dedi.txt");
-			std::string defaultArgs = "+setplaylist private_match";
```

Comment:
What is the behaviour for dedis when this functionality is lost?

Also, a number of dedis use `ns_startup_args_dedi.txt` to set up the server. If this gets merged we should warn server hosters about this no longer being supported